test/namedtuple.jl

```diff
@@ -276,3 +276,7 @@ let nt0 = NamedTuple(), nt1 = (a=33,), nt2 = (a=0, b=:v)
     @test Base.setindex(nt1, "value", :a) == (a="value",)
     @test Base.setindex(nt1, "value", :a) isa NamedTuple{(:a,),<:Tuple{AbstractString}}
 end
+
+# Pair constructor
+@test NamedTuple(:a => 1, :b => 2.0) === (a=1, b=2.0)
```

Comment:
do we also want to test `(; :a => 1, :b => 2.0)` and/or `(:a => 1, :b => 2.0)` ?

Reply:
I'll add in the former but the later is just a regular ole' `Tuple`